index.html

```diff
@@ -0,0 +1,193 @@
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Homework HTML/CSS</title>
+    <style>
+        html {
+            font-size: 15px;
+            font-weight: bold;
+        }
+
+        @media screen and (min-width: 320px) {
+            html {
+                font-size: calc(15px + 6 * ((100vw - 320px) / 680));
+            }
+        }
+        @media screen and (min-width: 1000px) {
+            html {
+                font-size: 22px;
+            }
+        }
+
+        body {
+            background-color: #ffeead;
+        }
+
+        .container {
+            display: grid;
+            grid-gap: 3px;
+            grid-template-columns: repeat(12, 1fr);
+            grid-template-rows: 50px auto 50px;
+            height: 100%;
+        }
+
+        .header {
+            grid-column: 2 / -1;
+
+            background-color: #ff706a;
+
+            display: flex;
+
+            padding: 15px;
+        }
+
+        .header > div:not(:last-child) {
+            margin-right: 15px;
+        }
+
+        .header > div:last-child {
+            margin-left: auto;
+        }
+
+        .logo {
+            grid-row: 1 / 2;
+            background-color: #96ceb4;
+        }
+
+        .menu {
+            grid-row: 2 / 3;
+
+            background-color: #00cc9a;
+
+            display: flex;
+            flex-direction: column;
+            padding: 10px;
+        }
+
+        @media only screen and (max-width: 450px) {
```

Comment:
(Code style suggestion) 
Try to keep code for same component in one place. For example before @media block you are styling .menu element and after @media, you continue to style .menu. This is not useful. Better finish with .menu first and then add other styling. This is a global suggestion. Keep element styles in same place.